### PR TITLE
Pull request for the project 

### PR DESCRIPTION
**Pull request by GraceAbike**

### CODE REVIEW

**Review by GraceAbike — COMMENTED** [review]
I have been fixing some error on the project

File: index.html [hunk 1]
```diff
@@ -1,44 +1,144 @@
...
+        <li><a href="https://angel.co/u/grace-abike"><img src="./images/Min.png" alt="Angel link"></a></li>
+      </ul>
+    <a class="btn" href="">Get my resume</a>
+  <div class="skill-list">
+  <ul class="language-list">
+    <li><a class="toggle-btn" href="#">Languages
+    <span class="arrow down"></span>
+    </a></li>
+    <br>
+    <li class="frame"><img src="images/Ellipse 1.png" alt="javascript image"> <span class="list">JavaScript</span></li>
```
Comment (isaacpitwa) [inline index.html:123]: - [ ] [OPTIONAL] it is useful to the readers to use descriptive file names for the images used in order to eliminate spaces. Spaces could lead to wrong directory paths.

File: index.html [hunk 2]
```diff
@@ -1,44 +1,144 @@
...
+    <li class="frame"><img src="images/Ellipse 1 (1).png" alt="html image"><span class="list">HTML</span></li>
+    <li class="frame"><img src="images/Ellipse 1 (3).png" alt="css image"><span class="list">CSS</span></li>
```
Comment (isaacpitwa) [inline index.html:125]: - [ ] [OPTIONAL] it is useful to the readers to use descriptive file names for the images used in order to eliminate spaces. Spaces could lead to wrong directory paths.

File: index.html [hunk 3]
```diff
@@ -1,44 +1,144 @@
...
+      </ul>
+    <a class="btn" href="">Get my resume</a>
+  <div class="skill-list">
+  <ul class="language-list">
+    <li><a class="toggle-btn" href="#">Languages
+    <span class="arrow down"></span>
+    </a></li>
+    <br>
+    <li class="frame"><img src="images/Ellipse 1.png" alt="javascript image"> <span class="list">JavaScript</span></li>
+    <li class="frame"><img src="images/Ellipse 1 (1).png" alt="html image"><span class="list">HTML</span></li>
```
Comment (isaacpitwa) [inline index.html:124]: - [ ] [OPTIONAL] it is useful to the readers to use descriptive file names for the images used in order to eliminate spaces. Spaces could lead to wrong directory paths.

File: index.html [hunk 4]
```diff
@@ -1,44 +1,144 @@
...
-  <footer>
-     <h3>LET’S CONNECT</h3>
-     <div class="contact">
-      <ul>
-        <li class="icon"><a href="https://twitter.com/GraceAbike1"><img src="./images/Twitt.png" alt="Twitter link"></a></li>
-        <li class="icon"><a href="https://www.linkedin.com/in/grace-abike-02770522a/"><img src="./images/Linkedin-icon.png" alt="Linkedin link"></a></li>
-        <li class="icon"><a href="#"><img src="./images/Minimax.png"></a></li>
-        <li class="icon"><a href="https://github.com/GraceAbike"><img src="./images/Git.png" alt="Github link"></a></li>
-        <li class="icon"><a href="https://angel.co/u/grace-abike"><img src="./images/Min.png" alt="Angel link"></a></li>
+      <h3>LET’S CONNECT</h3>
```
Comment (isaacpitwa) [inline index.html:26]: Great work achieving the About myself section design 👏🏽

- [ ] it is required that the About myself section design is achieved using flexbox on its contents. check out the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)

**Review by isaacpitwa — CHANGES_REQUESTED** [review]
Hi @GraceAbike 🙂 ,

You have done a tremendous job setting up your project. I really like it :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text


## Highlights 
- All linters are passing  ✅
- Correct use of GitHub flow ✅
- Well documented Repository ✅
-  Clean and readable code base ✅

## Required changes ♻️ 

Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @isaacpitwa.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (GraceAbike) [reply index.html:123]: Thank you 
i will make the changes

File: index.html (hunk 4 above)
Comment (GraceAbike) [reply index.html:26]: Flexbox its not required in About Myself section.
If you are taking about cards Yes you are right.
In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

**Review by mmsesay — COMMENTED** [review]
Hi @GraceAbike, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have linters setup.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mmsesay — APPROVED** [review]
Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - Good Readem :heavy_check_mark: 
 - No linter error :heavy_check_mark: 
 - Good PR title :heavy_check_mark: 
 - No PR summary :heavy_multiplication_x:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Please moving forward, it will be good to always provide a descriptive PR summary, explaining the changes that were introduced in your project. That will give other developers or visitors a brief of what you worked on.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**